File: python/codingbat/009.py
def not_string(str):
  if len(str) >= 3 and str[:3] == "not":
    return str
  return "not " + str
  # str[:3] goes from the start of the string up to but not
  # including index 3

print(not_string('candy')) #'not candy'
print(not_string('x')) #'not x'
print(not_string('not bad')) #'not bad'

File: home/old/python/009.cpp
#include <iostream>
#include <algorithm>
#include <string>
using namespace std;

string not_string(string a) {
    if (a.length()>=3 && !a.substr(0,3).find("not"))
        return a;
    else;
        return "not " + a;
}

int main() {
    cout << not_string("candy") << endl;
    cout << not_string("x") << endl;
    cout << not_string("not bad") << endl;
	return 0;
}
